lsp-bridge.el

```diff
@@ -519,6 +519,13 @@ you can customize `lsp-bridge-get-workspace-folder' to return workspace folder p
     (default                    . standard-indent)) ; default fallback
   "A mapping from `major-mode' to its indent variable.")
 
+(defcustom lsp-bridge-string-interpolation-open-chars-alist
+  '((python-mode . "[^\$]\{")
+    (yaml-mode . "\{\{")
+    (t . "\$\{"))
```

Comment:
The completion menu may popup in unintended situations if this part is not adjusted.

Reply:
I think we should remove `(t . "\$\{"))`, if some langugage need, then we add new rule.